fme.md

```diff
@@ -4,495 +4,532 @@ N/A means "not applicable" in this context
+[AnchoredSnapper](https://www.safe.com/transformers/anchored-snapper/)|Snap geometries to layer|Missing|Performance issues |
+[AngleConverter](https://www.safe.com/transformers/angle-converter/)|Missing|Needs expression functions to create circular strings
+[AngularityCalculator](https://www.safe.com/transformers/angularity-calculator/)|Missing|Needs expression functions to create circular strings
+[AppearanceExtractor](https://www.safe.com/transformers/appearance-extractor/) |Missing|Needs expression functions to create circular strings|Doesn't apply in QGIS|
+[AppearanceMerger](https://www.safe.com/transformers/appearance-merger/)|Missing|Needs expression functions to create circular strings|Doesn't apply in QGIS|
+[AppearanceRemover](https://www.safe.com/transformers/appearance-remover/)|Missing|Needs expression functions to create circular strings|Doesn't apply in QGIS|
+[AppearanceSetter](https://www.safe.com/transformers/appearance-setter/)|Missing|Needs expression functions to create circular strings|Doesn't apply in QGIS|
+[AppearanceStyler](https://www.safe.com/transformers/appearance-styler/)|Missing|Needs expression functions to create circular strings|Doesn't apply in QGIS|
+[ArcEstimator](https://www.safe.com/transformers/arc-estimator/)|Missing|N/A|Port from PostGIS? Partial work at https://github.com/nyalldawson/QGIS/tree/convert_to_curves|
+[ArcGISGridSnapper](https://www.safe.com/transformers/arcgis-grid-snapper/)|Missing|N/A
+[ArcGISOnlineConnector](https://www.safe.com/transformers/arcgis-online-connector/)|Missing|N/A
+[ArcPropertyExtractor](https://www.safe.com/transformers/arc-property-extractor/)|Missing|Needs expression functions to create circular strings
+[ArcPropertySetter](https://www.safe.com/transformers/arc-property-setter/)|Missing|Needs expression functions to create circular strings
+[ArcStroker](https://www.safe.com/transformers/arc-stroker/)|Segmentize by ...|Missing
+[AreaAmalgamator](https://www.safe.com/transformers/area-amalgamator/)|Missing|N/A
+[AreaBuilder](https://www.safe.com/transformers/area-builder/)|Polygonize|Missing
+[AreaCalculator](https://www.safe.com/transformers/area-calculator/)|Add geometry attributes|$area
+[AreaGapAndOverlapCleaner](https://www.safe.com/transformers/area-gap-and-overlap-cleaner/)|Snap geometries to layer|Missing
+[AreaOnAreaOverlayer](https://www.safe.com/transformers/area-on-area-overlayer/)|Union|N/A
+[AttributeCompressor](https://www.safe.com/transformers/attribute-compressor/) |Missing|N/A
+[AttributeCopier](https://www.safe.com/transformers/attribute-copier/)|Field calculator, Refactor fields|N/A
+[AttributeCreator](https://www.safe.com/transformers/attribute-creator/)|Add field to attributes table|N/A
+[AttributeDecompressor](https://www.safe.com/transformers/attribute-decompressor/)|Missing|N/A
+[AttributeDereferencer](https://www.safe.com/transformers/attribute-dereferencer/)|N/A |Needs expression functions to create circular strings
```

Comment:
There seems to be a lot of "Needs expression functions to create circular strings" incorrectly introduced in this commit?